src/components/TimerModal/TimerModal.js

```diff
@@ -0,0 +1,24 @@
+import ReactDOM from 'react-dom';
+import style from './Modal.module.scss';
```

Comment:
Not sure if we are using modules - i think we should just stick to standard .scss for consistency